### ISSUE TITLE
Question about the calculation of gap value.

### ISSUE THREAD

**Issue by ttbond** [mroosmalen/nanosv#19]

**Comment by ttbond** [comment]
Hi~
I met a new question about the calculation of gap value when I read the code.If two adjacent segments have different  orientations(which means that the values of (flag & 16) are different), the gap value is still (clips[i2] - clips[i]),but clips[i] and clips[i2] are at the different poles of template.

**Comment by ttbond** [comment]
I'm not sure if it still makes sense on this condition.

**Comment by jaesvi** [comment]
When parsing the segments, we take into account the orientation of the read (using the flag) to order the clipped portions of each segment (parse_bam.py Lines 60-77). Since we have already corrected the orientation there, it is not needed in the gap calculation. 
I made a quick drawing of what goes on that I hope clarifies things.
![img_20180323_142441](https://user-images.githubusercontent.com/14219434/37831538-2b8026a6-2ea6-11e8-8e8c-ff4517a51c7c.jpg)
Let me know,
Jose
